cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java

```diff
@@ -84,7 +97,7 @@ private MBeanServerConnection connect() {
     return mbsc;
   }
 
-  String nodeToString(Node node){
+  String nodeToString(Node node) {
     return String.format("%s:%d:%d", node.getIp(), node.getMetaPort(), node.getDataPort());
```

Comment:
I think we can also add `clientPort` here.

Reply:
ok